frontend/.changeset/famous-eyes-mix.md

```diff
@@ -0,0 +1,6 @@
+---
+"@liam-hq/erd-core": patch
+"@liam-hq/cli": patch
+---
+
+feat: Add cardinality comments to TableNode component
```

Comment:
I want the release notes to help users see the value in a straightforward way.
What about the following?

```suggestion
feat: 1:n and 1:1 notations can now be displayed when highlighting relationships
```

or

```suggestion
feat: Display 1:n and 1:1 labels when highlighting relationships
```

Reply:
Exactly! Thank you for your support.
Fixed: https://github.com/liam-hq/liam/pull/238/commits/853c380fd823f73da70fa91607dfe20d42a0688b